examples/msal-net/akvdotnet/TokenCredential.cs

```diff
@@ -22,9 +22,12 @@ public MyClientAssertionCredential()
         var tokenPath = Environment.GetEnvironmentVariable("AZURE_FEDERATED_TOKEN_FILE");
         var tenantID = Environment.GetEnvironmentVariable("AZURE_TENANT_ID");
 
-        _confidentialClientApp = ConfidentialClientApplicationBuilder.Create(clientID)
-                .WithClientAssertion(ReadJWTFromFS(tokenPath))
-                .WithTenantId(tenantID).Build();
+        _confidentialClientApp = ConfidentialClientApplicationBuilder
+                .Create(clientID)
+                .WithAuthority("https://login.microsoftonline.com/", tenantID) // for simplicity, this samples assume the app is in the public cloud
```

Comment:
The value for authority is available in` AZURE_AUTHORITY_HOST` environment variable.